docs/tools/compiler-toolchain/solidity.md

```diff
@@ -53,7 +53,7 @@ Additional zkEVM data inserted by `zksolc`:
 - `long_version`: the full `solc` version output.
 - `zk_version`: the `zksolc` version.
 - `contract/hash`: the hash of the zkEVM bytecode.
-- `contract/factory_dependencies`: bytecode hashes of contracts created in the current contract with `CREATE`. [More details here](../../reference/architecture/contract-deployment.md#note-on-factory_deps).
+- `contract/factory_dependencies`: bytecode hashes of contracts created in the current contract with `CREATE`. [More details here](../../reference/architecture/contract-deployment.md#note-on-factory-deps).
```

Comment:
This is actually incorrect, as the URL is supposed to have an underscore:

<img width="681" alt="image" src="https://github.com/matter-labs/zksync-web-era-docs/assets/1890113/617fa0a8-84b3-41e5-9022-5b05eb824fc7">

Reply:
Apologies, I was confusing GitHub links (looking at .md files directly) vs viewing from the hosted website. Closing my open conversations regarding this in the PR.